ui/packages/app/src/Blocks.svelte

```diff
@@ -457,31 +474,52 @@
 		{/if}
 	</div>
 	<footer
-		class="flex justify-center pb-6 text-gray-300 dark:text-gray-500 font-semibold"
+		class="flex justify-center pb-6 text-gray-400 space-x-2 text-sm md:text-base"
 	>
 		{#if show_api}
-			<div
-				class="cursor-pointer hover:text-gray-400 dark:hover:text-gray-400 transition-colors"
+			<button
 				on:click={() => {
-					api_docs_visible = !api_docs_visible;
+					set_api_docs_visible(!api_docs_visible);
 				}}
+				class="flex items-center hover:text-gray-500"
 			>
-				{#if api_docs_visible}hide{:else}view{/if} api
-			</div>
-			&nbsp; &bull; &nbsp;
+				View <img src={api_logo} alt="" class="w-2.5 md:w-3 mx-1" />API
+				documentation
```

Comment:
Probably going to have to rework a bit on this (not sure it's very clear)
<img width="613" alt="image" src="https://user-images.githubusercontent.com/3841370/201987062-8da5ad33-26f0-430a-b800-37ee5ff2f324.png">

Reply:
Agree, I think this should be tweaked so that the icons are not in the middle of the text. Perhaps "Use as API" instead of "View API documentation"?

Reply:
or maybe "Use this app API" to focus that it's *this* app's API

Reply:
(not sure)

Reply:
went with "Use via API", open to suggestions